.eslintrc.js

```diff
@@ -388,6 +387,13 @@ module.exports = {
          * typescript
          */
         "@typescript-eslint/no-empty-function": "off",
-        "@typescript-eslint/consistent-type-imports": "error",
+        "@typescript-eslint/no-this-alias": "off",
+        "@typescript-eslint/consistent-type-imports": [
+            "error",
+            {
+                prefer: "type-imports",
+                fixStyle: "separate-type-imports",
+            },
+        ],
```

Comment:
We've been discussing how it seems like this rule hasn't been working lately. I _think_ adding some configs has helped for this PR so we can see if this makes it work again (better).